src/main/clm_varctl.F90

```diff
@@ -230,6 +230,36 @@ module clm_varctl
 
   real(r8), public  :: o3_ppbv = 100._r8
 
+  ! number of wavelength bands used in SNICAR snow albedo calculation
+  integer, public :: snicar_numrad_snw = 5 
+
+  ! type of downward solar radiation spectrum for SNICAR snow albedo calculation
+  ! options:
+  ! mid_latitude_winter, mid_latitude_summer, sub_arctic_winter,
+  ! sub_arctic_summer, summit_greenland_summer, high_mountain_summer;
+  character(len=25), public :: snicar_solarspec = 'mid_latitude_winter'
```

Comment:
@ekluzek explained to me that the same issue applies to a few or several other sub-models, so this could be opened as an issue that addresses this one and another issue that addresses some or all of the others.

Reply:
link issue